sass/themes/frost/components/_form.scss

```diff
@@ -205,3 +206,10 @@ form {
     @extend .link--blue;
   }
 }
+
+.white__overlay {
```

Comment:
@lebaz20 I'd call this class something like `.form__overlay` for consistency with other classes and to avoid putting visual specifics in the name. Otherwise this makes sense to me.